packages/sdk/client/src/diagnostics/diagnostics.ts

```diff
@@ -25,6 +25,8 @@ import { Client } from '../client';
 import { getPlatform, Platform } from './platform';
 import { jsonStringify, JsonStringifyOptions } from './util';
 
+const DEFAULT_DIAGNOSTICS_TIMEOUT = 1000;
```

Comment:
too short. and always try to declare consts in the function's options property as a default that the caller (e.g., CLI) can override.